recipes/rnabob/removeman.patch

```diff
@@ -0,0 +1,10 @@
+--- Makefile	2016-09-18 01:24:06.664551971 +0200
++++ Makefile-new	2016-09-18 02:00:43.872926057 +0200
+@@ -58,7 +58,6 @@
+ 
+ install: $(PROGS)
+ 	cp $(PROGS)   $(BINDIR)/
+-	cp rnabob.man $(MANDIR)/man1/rnabob.1
```

Comment:
$PREFIX is a magic variable from the conda build system. It is the directory where you want to store your package. Nowhere else. So I really don't know what you want to do with $HOME. For more information please read the documentation: http://conda.pydata.org/docs/building/recipe.html